SUMMARY.md

```diff
@@ -9,19 +9,17 @@
   * [Business Model](company/about-mattermost/business-model.md)
   * [Mindsets](company/about-mattermost/mindsets.md)
 * ["How to" guides for staff](company/how-to-guides-for-staff/README.md)
+  * [How to use Expensify](company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify.md)
+  * [How to book airfare and travel](company/how-to-guides-for-staff/how-to-spend-company-money/airfare-and-traveling.md)
+  * [How to reimburse the company](company/how-to-guides-for-staff/how-to-spend-company-money/how-to-reimburse-the-company.md)
+  * [How to convert currencies](company/how-to-guides-for-staff/how-to-spend-company-money/currency-exchange-table.md)
   * [How to set up a 1-1 channel](company/how-to-guides-for-staff/how-to-setup-a-1-1-channel.md)
   * [How to update the handbook](company/how-to-guides-for-staff/how-to-update-handbook.md)
-  * [How to spend company money](company/how-to-guides-for-staff/how-to-spend-company-money/README.md)
-    * [How to use Expensify](company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify.md)
```

Comment:
Given these other pages are changes location in the TOC, e.g. how to use Expensify, old links will break

Reply:
That's weird - I'd removed them and then replaced them because I only wanted to test two links (spending company money and procuring a vendor). They're definitely still in the same place in the preview as far as I can tell.

Reply:
Oh wait - do you mean that moving the other two from the list also affects the remaining pages?

Reply:
In the new TOC, "how to use Expensify" will now be listed directly in the "how to guides" section, instead of in the 'how to spend company money' section. I think that will break the URL

Reply:
Oh I see what you mean, of course. It will definitely break the URL. I'll sort that out, thanks!